adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -31,21 +31,20 @@
 
 # pylint: disable=no-name-in-module
 
-import neopixel
 import adafruit_esp32spi
 import adafruit_esp32spi.adafruit_esp32spi_requests as requests
 
 class ESPSPI_WiFiManager:
     """
     A class to help manage the Wifi connection
     """
-    def __init__(self, esp, secrets, status_neopixel=None, attempts=2):
+    def __init__(self, esp, secrets, status_pixel=None, attempts=2):
         """
         :param ESP_SPIcontrol esp: The ESP object we are using
         :param dict secrets: The WiFi and Adafruit IO secrets dict (See examples)
         :param int attempts: (Optional) Failed attempts before resetting the ESP32 (default=2)
-        :param status_neopixel: (Optional) The neopixel pin - Usually board.NEOPIXEL (default=None)
-        :type status_neopixel: Pin
+        :param status_pixel: (Optional) The pixel pin - Usually board.NEOPIXEL (default=None)
```

Comment:
Why is Pin optional -- why would you pass in a pin name?

Reply:
Oh, I need to fix that.

Reply:
I'll have to wait until I get to my charger though as my laptop with my code died.

Reply:
Ok, I was able to use the online github editor to fix. The reason it is optional is because the library will function even if you don't pass it in. You just won't be able to visually tell the status.

Reply:
np with optional -- I was just confused but the Pin reference -- thanks for fixing. I like this because the user can use any neopixel/dotstar (single or string) available